x-pack/plugins/observability/public/components/shared/exploratory_view/header/add_to_case_action.tsx

```diff
@@ -40,7 +40,6 @@ export function AddToCaseAction({
 }: AddToCaseProps) {
   const kServices = useKibana<ObservabilityAppServices>().services;
   const userPermissions = useGetUserCasesPermissions();
-  const casesPermissions = { all: userPermissions.crud, read: userPermissions.read };
```

Comment:
Backing out the changes from my previous PR because all the `useGetUserCasesPermissions` return the same structure now.